tests/eas/load_tracking.py

```diff
@@ -0,0 +1,225 @@
+                    'tasks' : 1,
+                    'prefix' : 'fie_test',
+                    'cpus' : [cpu]
+                },
+            },
+        }
+
+        # Create a set of confs with different frequencies
+        # We'll run the 10% workload under each conf (i.e. at each frequency)
+        confs = []
+
+        all_freqs = test_env.target.cpufreq.list_frequencies(cpu)
+        # If we have loads of frequencies just test a cross-section so it
+        # doesn't take all day
+        cls.freqs = all_freqs[::len(all_freqs)/8 + 1]
+        for freq in cls.freqs:
+            confs.append({
+                'tag' : 'freq_{}'.format(freq),
+                'flags' : ['ftrace', 'freeze_userspace'],
+                'cpufreq' : {
+                    'freqs' : {cpu: freq},
+                    'governor' : 'userspace',
+                },
+            })
+
+        return {
+            'wloads': wloads,
+            'confs': confs,
+        }
+
+    def get_expected_util_avg(self, experiment):
+        """
+        Examine trace to figure out an expected mean for util_avg
+
+        Assumes an RT-App workload with a single task with a single phase,
+        running on a CPU with the highest max capacity in the system
+
+        This takes into account the frequency the workload was run at, but
+        doesn't use the kernel's data for compute capacities at each frequency,
+        instead it assumes that these values scale linearly.
+        """
+        [task] = experiment.wload.tasks.keys()
+        sched_assert = self.get_sched_assert(experiment, task)
+
+        [freq] = experiment.conf['cpufreq']['freqs'].values()
+        freq_scaling_factor = float(freq) / max(self.freqs)
+        duty_cycle_pct = sched_assert.getDutyCycle(self.get_window(experiment))
+
+        return UTIL_SCALE * (duty_cycle_pct / 100.) * freq_scaling_factor
+
+    def get_sched_task_signals(self, experiment, signals):
+        """
+        Get a pandas.DataFrame with the sched signals for the workload task
+
+        This examines scheduler load tracking trace events, supporting either
+        sched_load_avg_task or sched_pelt_se. You will need a target kernel that
+        includes these events.
+
+        :param experiment: Experiment to get trace for
+        :param signals: List of load tracking signals to extract. Probably a
+                        subset of ``['util_avg', 'load_avg']``
+        :returns: :class:`pandas.DataFrame` with a column for each signal for
+                  the experiment's workload task
+        """
+        [task] = experiment.wload.tasks.keys()
+        trace = self.get_trace(experiment)
+
+        # There are two different scheduler trace events that expose the load
+        # tracking signals. Neither of them is in mainline. Eventually they
+        # should be unified but for now we'll just check for both types of
+        # event.
+        # TODO: Add support for this parsing in Trappy and/or tasks_analysis
```

Comment:
Ok, let's progress this 17.02 and fix/update this later.
At the very least, we will have automated tests remembering us about broken events. ;-)